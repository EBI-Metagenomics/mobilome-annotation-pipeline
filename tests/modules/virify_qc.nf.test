nextflow_process {

    name "Test Process VIRIFY_QC"
    script "modules/virify_qc.nf"
    process "VIRIFY_QC"

    test("Should process run virify_qc successfully") {

        when {
            params {
                outdir = "tests/results"
            }
            process {
                """
                input[0] = file("${projectDir}/tests/test_data/virify_example/virify.gff")
                input[1] = file("${projectDir}/tests/test_data/virify_example/prophages_quality_summary.tsv")
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
