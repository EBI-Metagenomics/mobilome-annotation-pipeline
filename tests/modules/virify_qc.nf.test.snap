{
    "Should process run virify_qc successfully": {
        "content": [
            {
                "0": [
                    "virify_hq.gff:md5,27f8d1cc41da488c6f212e9c2d10b9a8"
                ],
                "virify_hq": [
                    "virify_hq.gff:md5,27f8d1cc41da488c6f212e9c2d10b9a8"
                ]
            }
        ],
        "timestamp": "2023-11-28T17:01:43.022476"
    }
}